src/_listByline.scss

```diff
@@ -37,7 +37,7 @@
       border-radius: $fd-list-byline-thumbnail-border-radius;
       margin-right: $fd-list-byline-thumbnail-spacing;
       font-size: 2.5rem;
-      align-self: flex-start;
+      align-self: center;
```

Comment:
here icon/avatar should be centre align, https://wiki.wdf.sap.corp/wiki/display/visualcore/List+%28Fiori+3%29+-+Standard+List+with+Byline.
Borderless byline has issue with flex-start.

Reply:
You can remove this rule, you don't need to change it to center.